rollup.config.mjs

```diff
@@ -97,7 +98,20 @@ function bundleConfig({ name, entry }) {
 }
 
 export default [
-  bundleConfig({ name: 'annotator', entry: 'src/annotator/index.js' }),
+  bundleConfig({
+    name: 'annotator',
+    entry: 'src/annotator/index.js',
```

Comment:
OK, this makes sense. Only the annotator bundle needs to be `iife` as we have control over the sidebar frame's MIME- and document-types.